docs/GraFx-Studio/guides/output/settings/index.md

```diff
@@ -1,11 +1,108 @@
----
-tags:
-  - unfinished
----
+# Create Output Settings
 
-# Output Settings
+[What are output settings?](/GraFx-Studio/concepts/output-settings/)
 
-Placeholder to describe where and how to create output settings.
-[See also description](https://chilipublishintranet.atlassian.net/wiki/spaces/dev/pages/3806691367/GraFx+Studio+MVP+Animation+export+settings){target="_blank"}
+## Create output settings
 
-(link will be removed, once content is ready)
\ No newline at end of file
+In the GraFx Studio application, under manage, you can manage "Output settings"
+
+To add a setting click the "+ create" button.
```

Comment:
By default, there will be default output settings for each output format. The "No output settings created" placeholder will only be visible if you delete all default settings.
So, maybe we don't need a screenshot of that in the docs and it would be good to mention that we already provide our users with default settings, but you can change or remove them or add more if you want.

Reply:
Moved it to where we explain deleting settings